wfe/context.go

```diff
@@ -13,19 +13,17 @@ import (
 )
 
 type requestEvent struct {
-	ID            string `json:",omitempty"`
-	RealIP        string `json:",omitempty"`
-	ClientAddr    string `json:",omitempty"`
-	Endpoint      string `json:",omitempty"`
-	Method        string `json:",omitempty"`
-	Errors        []string
-	Requester     int64                  `json:",omitempty"`
-	Contacts      *[]string              `json:",omitempty"`
-	RequestNonce  string                 `json:",omitempty"`
```

Comment:
I've used the `RequestNonce` and `ResponseNonce` in the past to help folks diagnose their bad nonce errors. It doesn't happen often (once a month? less?) but it has been useful on a handful of occasions. Rather than remove them entirely could we print the nonces truncated? Maybe `[0..3]...[-3:]` (e.g. the first 3 & last 3 chars)?